backend/app/router.js

```diff
@@ -206,6 +206,9 @@ router.route('/:realm/v0.2/tasks/:id')
     .put(authenticate('jwt').always, jsonParser, tasks.updateOne)
     .delete(authenticate('jwt').always, tasks.delete);
 
+router.route('/:realm/v0.2/tasks-with-discussions/:id')
```

Comment:
Okay, so we actually need this to return all the tasks with the discussion stuff done here, for a project. I can work with it being for all projects or a specific one.